From bb4d440337258ea8a047779c9ca4875c73806ee7 Mon Sep 17 00:00:00 2001
From: Gao Xiang <xiang@kernel.org>
Date: Wed, 18 Feb 2026 21:13:51 +0800
Subject: [PATCH] erofs-utils: manpage: fix installation of `erofsfuse.1`

`erofsfuse.1` may be missing if `man_MANS` is assigned twice.

Forwarded: https://lore.kernel.org/r/20260218131351.46839-1-xiang@kernel.org
Origin: upstream, https://git.kernel.org/xiang/erofs-utils/p/bb4d44033725
Signed-off-by: Gao Xiang <xiang@kernel.org>
---
 man/Makefile.am | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index c951681..b9b5989 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -3,10 +3,12 @@
 dist_man_MANS = mkfs.erofs.1 dump.erofs.1 fsck.erofs.1
 
 EXTRA_DIST = erofsfuse.1 mount.erofs.8
+
+man_MANS =
 if ENABLE_FUSE
-man_MANS = erofsfuse.1
+man_MANS += erofsfuse.1
 endif
 
 if OS_LINUX
-man_MANS = mount.erofs.8
+man_MANS += mount.erofs.8
 endif
-- 
2.47.3

